docs/src/main/asciidoc/mp/guides/oci-guide.adoc

```diff
@@ -0,0 +1,178 @@
+│   ├── README.md
+│   ├── app.yaml
+│   ├── distribution.xml
+│   ├── pom.xml
+│   └── src
+│       ├── main
+│       │   ├── java
+│       │   │   └── me
+│       │   │       └── username
+│       │   │           └── mp
+│       │   │               └── oci
+│       │   │                   └── server
+│       │   │                       ├── GreetLivenessCheck.java
+│       │   │                       ├── GreetReadinessCheck.java
+│       │   │                       ├── GreetResource.java
+│       │   │                       ├── GreetingProvider.java
+│       │   │                       └── package-info.java
+│       │   └── resources
+│       │       ├── META-INF
+│       │       │   ├── beans.xml
+│       │       │   ├── microprofile-config-prod.properties
+│       │       │   ├── microprofile-config-test.properties
+│       │       │   └── microprofile-config.properties
+│       │       ├── application.yaml
+│       │       └── logging.properties
+│       └── test
+│           └── java
+│               └── me
+│                   └── username
+│                       └── mp
+│                           └── oci
+│                               └── server
+│                                   ├── Common.java
+│                                   ├── GreetResourceConfigFileTest.java
+│                                   ├── GreetResourceInstancePrincipalTest.java
+│                                   └── GreetResourceMockedTest.java
+└── spec
+    ├── README.md
+    └── api.yaml
+----
+====
+
+Alternatively, you can also use the Helidon OCI Maven archetype or the Helidon CLI to generate the Helidon MP OCI project instead.
+
+== Running the Application Locally
+
+Use the project to build an application jar for the example. It saves all runtime dependencies in the `/target/libs` directory, allowing you to easily start the application by running the application jar file.
+
+. Change to the directory containing the project files and run:
+
+[source,bash]
+.Build the Application
+----
+mvn package
+----
+
+. Then, to run the application, run:
+
+[source,bash]
+.Run the application
+----
+java -jar server/target/myproject-server.jar
+----
+
+The example is a very simple "Hello World" greeting service. It supports GET requests for generating a greeting message, and a PUT request for changing the greeting itself. The response is encoded using JSON.
+
+For example:
+
+[source,bash]
+.Try the Application
+----
+curl -X GET http://localhost:8080/greet
+{"message":"Hello World!"}
+
+curl -X GET http://localhost:8080/greet/Joe
+{"message":"Hello Joe!"}
+
+curl -X PUT -H "Content-Type: application/json" -d '{"greeting" : "Hola"}' http://localhost:8080/greet/greeting
+
+curl -X GET http://localhost:8080/greet/Jose
+{"message":"Hola Jose!"}
+----
+
+== Deploying a Helidon Application on OCI
+
+You can deploy the Helidon application on either OCI OKE or OCI Compute. Depending on your requirements, one option might be better suited to your needs than the other.
```

Comment:
For OKE, use the full name of the service first i.e. Oracle Kubernetes Engine before using the acronym.